resources/netlog_defs.ts

```diff
@@ -759,4 +814,21 @@ const logDefinitions = {
 const assertLogDefinitions: LogDefinitionMap = logDefinitions;
 console.assert(assertLogDefinitions);
 
+export type LogDefinitions = typeof logDefinitions;
+export type LogDefinitionTypes = keyof LogDefinitions;
```

Comment:
I don't think this is really a singular type though? This is more akin to the existing `NetFields` from main, e.g. this could also be defined as:

```ts
export type LogDefinitions = {
  'GameLog': typeof logDefinitions['GameLog'];
  'HeadMarker': typeof logDefinitions['HeadMarker'];
...
```

So it's not really a singular type, it's a map of types. Also, `LogDefinition` is already a defined type at the top of the file:
```ts
export type LogDefinition = {
  type: string;
  name: string;
...
```

Reply:
Hmm, fair.  Ok!